airbyte-integrations/connectors/source-zendesk-chat/metadata.yaml

```
@@ -28,6 +29,11 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "The subdomain is mandatory as per the recent update for base url: `{config['subdomain']}.zendesk.com/api/v2/chat/`"
+        upgradeDeadline: "2024-10-29"
```

```suggestion
        upgradeDeadline: "2024-11-11"
```

The deadline should  at least be 1 week in the future ([ref](https://docs.airbyte.com/contributing-to-airbyte/resources/qa-checks#breaking-change-deadline-should-be-a-week-in-the-future))